### PR TITLE
Fix 3D objects not being displayed in example of  `save_graphic`

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

resolves https://github.com/pyvista/pyvista/discussions/4584

| Main  | PR |
| ------------- | ------------- |
| image |  image |

### Details

- None

### CODE REVIEW

**Comment by adeak** [comment]
Which change fixes the issue? Is it the render step that was missing before saving? We might consider pointing that out in the examples, or even making it an optional feature of the export.

**Comment by tkoyama010** [comment]
I began to think that it would be user-friendly to follow the screenshot approach. Please give me some time to consider this.
https://github.com/pyvista/pyvista/blob/65042bc9c1f9b0c9b1722c1dca74fcbd862a4b25/pyvista/plotting/plotter.py#L5714-L5720

File: pyvista/plotting/plotter.py [hunk 1]
```diff
@@ -5631,10 +5631,12 @@ def save_graphic(self, filename, title='PyVista Export', raster=True, painter=Tr
         --------
         >>> import pyvista
         >>> from pyvista import examples
-        >>> pl = pyvista.Plotter()
+        >>> pl = pyvista.Plotter(off_screen=True)
```
Comment (banesullivan) [inline pyvista/plotting/plotter.py:5634]: I'm hoping this `off_screen` argument isn't necessary and that we are setting `pyvista.OFF_SCREEN` when running the docstrings.

Comment (tkoyama010) [reply pyvista/plotting/plotter.py]: Good point. thanks!

**Comment by tkoyama010** [comment]
Rendering during first-time rendering solved this issue.
img

**Review by banesullivan — APPROVED** [review]

**Pull request merged**